var/spack/repos/builtin/packages/z/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```

Comment:
Please remove the bolierplate (lines 6-21) and cleanup FIXME comments

Reply:
Thanks, I've fixed it. That folder was supposed to be deleted